addons/crm/models/crm_lead.py

```diff
@@ -2027,6 +2027,9 @@ def _message_post_after_hook(self, message, msg_vals):
                 lambda partner: partner.email == self.email_from or (self.email_normalized and partner.email_normalized == self.email_normalized)
             )
             if new_partner:
+                customer_info = self._get_customer_information().get(self.email_normalized or self.email_from, {})
+                if customer_info and new_partner[0].email == new_partner[0].name != customer_info.get('name'):
+                    new_partner[0].update(customer_info)
                 if new_partner[0].email_normalized:
```

Comment:
Have to say this is weird ... I did not follow the other PR in details, but the newly created partner should be created with the right values, not created then updated afterwards because we did not achieve to control the values we give to the frontend. I think the whole suggested recipients thingy is getting ridiculous, we just want to propagate 5 values when creating a partner ...